src/Utilities/MoistThermodynamics/MoistThermodynamics.jl

```diff
@@ -253,7 +253,7 @@ internal_energy(ts::PhaseEquil) = ts.e_int
 internal_energy(ts::PhaseNonEquil) = ts.e_int
 
 """
-    internal_energy_sat(T, ρ, q_tot)
+    internal_energy_sat(T, ρ, q_tot) #TODO - rename or change documentation (this is misleading)
```

Comment:
What is misleading about it?

Reply:
It's just internal energy and not internal energy at saturation. Or in other words it will only return internal energy at saturation if the input temperature provided will be at saturation. Which is misleading to me.

Reply:
The temperature, density, and q_tot define the equilibrium point in the phase diagram. Seems standard and ok to me. (You need 3 variables to define a thermodynamic equilibrium state.)

Reply:
I agree with this. But if as a user I don't make sure that my 3 input variables are at equilibrium then this function will not return internal energy at saturation?

Reply:
Any 3 input variables **define** an equilibrium point.

Reply:
Sorry. I was thinking in terms of the equilibrium I want to reach when doing saturation adjustment. I'll remove the todo.